brainstormforce/ultimate-addons-for-gutenberg
# Fix: FAQ - Updated Row gap value same as frontend.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/3sAgbH1y7qY/581c0872-740c-4c3b-85eb-310388ce35db?redirect=true&view=6e170ea7-8708-4546-8b82-23d2eebdf694&t=attributes
- Updated row gap value same as frontend.

### Types of changes




- Updation

### How has this been tested?

- Drag and drop FAQ.
- Check row gap on the frontend and editor it should be the same.

### Checklist:
